packages/template-mrt-reference-app/app/ssr.js

```diff
@@ -263,6 +274,7 @@ const {handler, app, server} = runtime.createHandler(options, (app) => {
     app.get('/cookie', cookieTest)
     app.get('/headers', headerTest)
     app.get('/isolation', isolationTests)
+    app.get('/response-header', responseHeader)
```

Comment:
Would `/set-response-header` be a more descriptive name for this endpoint / handler?